<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Launcher</class>
 <widget class="QDialog" name="Launcher">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>프로젝트 리스트</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QListView" name="ui_ProjectList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>경로</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ui_WorkingDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_FindWorkingDir">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="ui_OpenProject">
         <property name="text">
          <string>프로젝트 열기 (&amp;O)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_NewProject">
         <property name="text">
          <string>새로운 프로젝트 (&amp;N)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ui_Update">
         <property name="text">
          <string>업데이트 (&amp;U)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_Close">
         <property name="text">
          <string>종료 (X)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ui_ProjectList</tabstop>
  <tabstop>ui_WorkingDir</tabstop>
  <tabstop>ui_FindWorkingDir</tabstop>
  <tabstop>ui_OpenProject</tabstop>
  <tabstop>ui_NewProject</tabstop>
  <tabstop>ui_Update</tabstop>
  <tabstop>ui_Close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
